content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx

```diff
@@ -79,7 +79,7 @@ Consider using nested writes if:
 
 <Tip>
 
-If you [pre-compute your IDs, you can choose between a nested write or using the `$transaction` API](#scenario-pre-computed-ids-and-the-transaction-api).
+If you [pre-compute your IDs, you can choose between a nested write or using the `$transaction([])` API](#scenario-pre-computed-ids-and-the-transaction-api).
```

Comment:
Q: Would referring to `$transaction([])` as sequential transaction operations make sense? If so, I would be happy to take care of the updates 🤔

Example usage: 
```suggestion
If you [pre-compute your IDs, you can choose between a nested write or using sequential transaction operations](#scenario-pre-computed-ids-and-the-transaction-api).
```

Reply:
A further rewrite of these two pages is definitely justified, potentially including your suggestion. I created https://github.com/prisma/docs/issues/5123 for the follow up work here.

Reply:
Could we still approve and merge this in the meantime?

Reply:
I've created a new GH issue for further updates to our transaction guides.